dependencies/bom/pom.xml

```diff
@@ -365,6 +365,12 @@
         <artifactId>transport-rest-vertx</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <!-- ServiceComb third-extensions -->
+      <dependency>
+        <groupId>org.apache.servicecomb</groupId>
+        <artifactId>cas-env-variables</artifactId>
```

Comment:
suggestions:

third-party-extension/huaweicloud,  do not use cas.

Reply:
done, use huawei-cloud/servicestage